coderd/wsbuilder/wsbuilder.go

```diff
@@ -632,19 +637,34 @@ func (b *Builder) getParameters() (names, values []string, err error) {
 		//  conditional parameter existence, the static frame of reference
 		//  is not sufficient. So assume the user is correct, or pull in the
 		//  dynamic param code to find the actual parameters.
+		latestValues := make(map[string]string)
```

Comment:
```suggestion
		latestValues := make(map[string]string, len(b.richParameterValues))
```

For maps this isn't capacity, just the initial allocation